### ISSUE TITLE
Error on qr-code-styling package

### ISSUE THREAD

**Issue by perfogic** [nomic-io/nomic-bitcoin-js#3]
Hello, i'm testing your package but i face this error:
![image](https://github.com/nomic-io/nomic-bitcoin-js/assets/74671798/eb4c2ee2-6d95-48ef-9a09-bb9a417a796f)

After finding the error, they said that, it can't be used in backend environment. I just wonder how you guys use it, whether any settings that i missed are existed ?

**Comment by blake-regalia** [comment]
Any updates from the core team? This prevents the library from being used in node.js. A simple fix would be to remove the qrcode import from `index.ts` and allow user's to import the QR stuff through ESM if they need/want it.

**Comment by perfogic** [comment]
yep, current fix is removing the qrcode creation in node development.

**Comment by blake-regalia** [comment]
Why was this issue closed? There is no workaround, so i'm not sure what you mean by "current fix"? @perfogic

**Comment by perfogic** [comment]
yeah, you have to change the code. I closed because i don't see any responses from the team
